### PR TITLE
Added Movie Finder App

### PR DESCRIPTION
**Pull request by khushi-purwar**
### Description

A single page web application that allows users to search movies based on their query.

### Checklist

- [x] I've been assigned an issue related to this PR.
- [x] I've used beautifiers.
- [x] I've added my Project's name and description to `Index.md`
- [x] I've made a README.md file for my Project.
- [x] The README.md file of my projrct contains Project title, Description, Use of project, Set up, Stack used and Output (Screenshots). 

## Related Issues or Pull Requests number

#281 

ss2

### CODE REVIEW

**Review by harshita2216 — CHANGES_REQUESTED** [review]
https://www.codefactor.io/repository/github/ayushparikh-code/web-dev-mini-projects/pull/289

Kindly make the following changes.

**Review by harshita2216 — APPROVED** [review]

File: Index.md [hunk 1]
```diff
@@ -109,3 +109,6 @@
 | [Counter APP](https://github.com/soma2000-lang/Web-dev-mini-projects/tree/iron/counter_App) |It is a simple react app which is used to incere or decrease the value of counter.
 | [BMI Calculator (JS)](https://github.com/soma2000-lang/Web-dev-mini-projects/tree/iron/BMI%20Calculator%20(JS)) |Calculates the Body Mass Index of a person, provided with a BMI chart.
 | [React Movie Finder Application](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/react-movie-app) |A single page web application that allows users to search movies based on their query.
+| [Maths Quiz Game](https://github.com/nisha331/Web-dev-mini-projects/tree/main/Maths%20quiz%20game) | A maths quiz game for kids to practice addition.
+| [Personal Blog Template](https://github.com/soma2000-lang/Web-dev-mini-projects/tree/iron/Personal-blog) |A simple and responsive personal blog template.
```
Comment (harshita2216) [inline Index.md:113]: I was trying to resolve this on my mobile but was unable to do it.  😂 Thanks for doing this.

Comment (khushi-purwar) [reply Index.md:113]: welcome 😊

**Comment by khushi-purwar** [comment]
Kindly provide the tag for this PR @harshita2216

**Comment by harshita2216** [comment]
> Kindly provide the tag for this PR @harshita2216

Thanks @khushi-purwar. Kindly check some other PRs which doesn't have labels. DM me their links on Discord please?

**Comment by khushi-purwar** [comment]
> > Kindly provide the tag for this PR @harshita2216
> 
> Thanks @khushi-purwar. Kindly check some other PRs which doesn't have labels. DM me their links on Discord please?

ok

**Comment by khushi-purwar** [comment]
> > Kindly provide the tag for this PR @harshita2216
> 
> Thanks @khushi-purwar. Kindly check some other PRs which doesn't have labels. DM me their links on Discord please?

Hey, @harshita2216 All the PR's having their tags.

**Pull request merged**